### DISCUSSION TITLE
Can you set calendar-specific color?

### DISCUSSION THREAD

**Discussion by modo1302** [totaldebug/atomic-calendar-revive#1213] [Support] [answered]
Hey! is it possible to set event colors (or calendar-name colors) based on a specific calendar entity?
example: event from my calendar - blue, event from my partner calendar - pink?

I'm using event mode.
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/133964285/9d464b49-6baf-49c9-ac13-b2cdcfdb1aaf)

**Comment by marksie1988** [comment, marked as answer]
Yes, under the entity add `color` and the colour you want

**Reply by modo1302** [reply]
Works flawlessly, thanks!
